lib/be_protected/response/verification/rules.rb

```diff
@@ -22,14 +22,22 @@ def passed?
           true
         end
 
+        def has_action?(action)
+          attributes.keys.each do |account|
```

Comment:
I suppose this piece can be written in more simple way:
```Ruby
attributes.values.each do |account_bucket|
  account_bucket.values.each do |alias_name_bucket|
    return true if alias_name_bucket.values.include?(action)
  end
end
```
p.s.: I don't know the domain, therefore, I've picked not enough reasonable names for hash values. Please, change names if you can.